launcher/src/store/services.js

```diff
@@ -2224,6 +2224,37 @@ export const useServices = defineStore("services", {
             network: "",
           },
         },
+        {
+          id: 35,
+          name: "SSV Monitoring",
+          service: "SSVNOMService",
+          displayPluginMenu: false,
+          serviceIsPending: false,
+          modifierPanel: false,
+          replacePanel: false,
+          addPanel: false,
+          configPanel: false,
+          category: "service",
+          path: "/ssvnoms",
+          icon: "/img/icon/service-icons/Other/MetricsExporter.png",
+          sIcon: "/img/icon/service-icons/Other/MetricsExporter-s.png",
```

Comment:
@daGscheid icon needed :^)

Reply:
Here you go :)

![Blox SSV Network Monitoring Service](https://github.com/user-attachments/assets/79e2d23f-e117-4a91-b5e1-d1a94bc2f19a)
![Blox SSV Network Monitoring Service - Service - Circle](https://github.com/user-attachments/assets/3b6f8912-fd6c-47f5-a38e-5add4fc00904)